examples/spouse_example.ddl

```diff
@@ -18,17 +20,19 @@ people_mentions(
   text           text,
   mention_id     text).
 
-has_spouse_candidates(
+has_spouse(
   person1_id  text,
   person2_id  text,
   sentence_id text,
   description text,
-  relation_id text).
+  relation_id text,
+  is_true boolean).
+
 has_spouse_features(
   relation_id text,
   feature     text).
 
-has_spouse?(relation_id text).
+f_has_spouse?(relation_id text).
```

Comment:
I think @feiranwang and I deliberately chose the names `X_candidates` and `X` to make it clearer as we inverted the variable/factor declaration in the compiled output.